pkg/sshutil/sshutil.go

```diff
@@ -231,7 +237,7 @@ func SSHOpts(instDir string, useDotSSH, forwardAgent, forwardX11, forwardX11Trus
 	if err != nil {
 		return nil, err
 	}
-	opts, err := CommonOpts(useDotSSH)
+	opts, err := CommonOpts(useDotSSH, hostAddress == "127.0.0.1")
```

Comment:
What about `::1`?

Reply:
It was only meant to handle the default string, not any overrides like "localhost" or "::1" (ipv6)

There is probably a utility `net` function that one can call, instead of hardcoding strings like this.

Reply:
https://pkg.go.dev/net#IP.IsLoopback should be used